readme.md

```diff
@@ -116,6 +116,20 @@ NOTE: specifying an output path with multiple formats ignores your specified ext
 * `./<HOST>_<DATE>.report.html`
 * `./<HOST>_<DATE>.artifacts.log`
 
+### Testing on a site with authentication
+
+If you have installed Lighthouse globally 
+
+ - `chrome-debug`
+ - open and login to your site
+ - `lighthouse http://mysite.com`
+ 
+If you use repository
```

Comment:
lets get rid of this second part. 
also you can just `npm link .` to install the repo globally. :)